cmd/explain.go

```diff
@@ -99,11 +96,12 @@ func getNetworkDetails(network *net.IPNet) *networkDetailsToDisplay {
 	// Obtain the total count of addresses in the network.
 	count := core.GetAddressCount(network)
 	// Format the count as a human-readable string and store it in the details struct.
-	if count.Cmp(big.NewInt(math.MaxInt64)) == 1 {
-		details.Count = message.NewPrinter(language.English).Sprintf("%s", count.String())
-	} else {
-		details.Count = message.NewPrinter(language.English).Sprintf("%d", count.Uint64())
-	}
+	details.Count = helper.FormatNumber(count.String())
```

Comment:
FormtNumber seems undefined

Reply:
That's my  bad, been git committing -a, forgot it doesn't factor in NEW files. I'll push that real quick.